app/controllers/provider/passwords_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Provider::PasswordsController < FrontendController
 
-  skip_before_action :login_required, only: %i(destroy show update)
+  skip_before_action :login_required, only: %i(destroy show update request_password_reset)
```

Comment:
Change `(...)` for `[...]` plz 😄 

![image](https://user-images.githubusercontent.com/11318903/76544335-23f7e580-6488-11ea-9710-8df7b4bd4bd9.png)